keps/sig-scheduling/3243-respect-pod-topology-spread-after-rolling-upgrades/README.md

```diff
@@ -659,13 +665,18 @@ feature flags will be enabled on some API servers and not others during the
 rollout. Similarly, consider large clusters and how enablement/disablement
 will rollout across nodes.
 -->
+It won't impact already running workloads because it is an opt-in feature.
```

Comment:
It won't impact already running workloads because it's a scheduler feature - if something is already running, scheduler won't impact it for sure.

What about potential rollout issues?

Reply:
Updated